classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -51,7 +51,7 @@ public function run()
             $config['channel'] = $request['channel'];
             $config['archive.filename'] = Upgrader::DEFAULT_FILENAME;
         }
-        if (isset($request['private_release_link']) && isset($request['private_release_md5'])) {
+        if (isset($request['private_release_link'], $request['private_release_md5'])  ) {
```

Comment:
You add useless spaces :)

Reply:
Thank you @PierreRambaud. I fixed this part in my script. :)